autoscrub/scripts/cli.py

```diff
@@ -23,8 +23,10 @@
 import autoscrub
 import click
 import requests
+from distutils.version import LooseVersion
+from version_check import get_ffmpeg_version

```

Comment:
I think the version_check import statement is wrong? Probably should be from autoscrub.version_check? Pretty sure it will fail depending on the terminal cwd at the moment.

Could consider importing the version check module into `autoscrub/__init__.py` so it's available via the `import autoscrub` line above without needing another specific import line.

Reply:
Good pickup. Deliberately put utility function in a separate in the interest of not perpetuating monolithic __init__.py, but perhaps that train has left the station. I also now only check the version for those commands that are known to be incompatible with latest ffmpeg.